packages/@best/utils/src/__tests__/req.spec.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+*/
+
+import path from 'path';
+import { req } from '../req';
+
+describe('req', () => {
+    it('returns the default export for an es5 module', () => {
```

Comment:
Let's be [consistent](https://github.com/salesforce/best/search?q=should) with what already exists.

```suggestion
    it('should return the default export for an ES5 module', () => {
```